src/types/__tests__/profile.test.ts

```diff
@@ -84,6 +85,9 @@ describe("profile type", () => {
     expect(userData.family_name).toBe(mockedUser.family_name);
     expect(userData.fiscal_code).toBe(mockedUser.fiscal_code);
     expect(userData.has_profile).toBeTruthy();
+    expect(userData.is_email_enabled).toBe(
+      fromNullable(mockedExtendedProfile.is_email_enabled).getOrElse(true)
```

Comment:
`toBe(true)` ? or` toBe(anIsInboxEnabled)` (it makes more clear what's the expected output)

Reply:
Fixed in 6df8b41